src/paramTypes.ts

```diff
@@ -18,8 +18,18 @@ export type DataObjectType = {
 	};
 }
 
+export type queueObjectType = {
+	name: string,
+	middleware?: middlewareFunc, 
+};
+
+export type middlewareFunc = 
```

Comment:
```suggestion
export type MiddlewareFuncType = 
```
types should begin with upper-case and end with 'Type'

Reply:
done